### PR TITLE
Update example and README

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ The `bluerobotics-ping` package installs a `simplePingExample.py` script to get
 
 `$ simplePingExample.py --device <your-device>`
 
+It's also possible to connect via UDP server using the `--server` option with IP:PORT as input (e.g `192.168.2.2:9090` or `0.0.0.0:12345`).
```
Comment (Williangalvani) [inline README.md]:
```suggestion
It's also possible to connect via UDP server using the `--udp` option with IP:PORT as input (e.g `192.168.2.2:9090`).
```

File: examples/simplePingExample.py [hunk 2]
```diff
@@ -11,12 +11,22 @@
 ############################
 
 parser = argparse.ArgumentParser(description="Ping python library example.")
-parser.add_argument('--device', action="store", required=True, type=str, help="Ping device port.")
-parser.add_argument('--baudrate', action="store", type=int, default=115200, help="Ping device baudrate.")
+parser.add_argument('--device', action="store", required=False, type=str, help="Ping device port. E.g: /dev/ttyUSB0")
+parser.add_argument('--baudrate', action="store", type=int, default=115200, help="Ping device baudrate. E.g: 115200")
+parser.add_argument('--server', action="store", required=False, type=str, help="Ping UDP server. E.g: 0.0.0.0:12345")
```
Comment (Williangalvani) [inline examples/simplePingExample.py]:
```suggestion
parser.add_argument('--udp', action="store", required=False, type=str, help="Ping UDP server. E.g: 192.168.2.2:12345")
```

**Comment by Williangalvani** [comment]
my first impression when I saw `--server` was that this was going to act as a server. I think it needs a better name.
"0.0.0.0" is also something that only works on server side, and didn't work here (maybe it tried localhost or literal 0.0.0.0? idk).
Other than that it looks good

File: README.md (hunk 1 above)
Comment (patrickelectric) [reply README.md]: Done.

File: examples/simplePingExample.py (hunk 2 above)
Comment (patrickelectric) [reply examples/simplePingExample.py]: Done.

**Comment by patrickelectric** [comment]
@Williangalvani I also moved the others scripts from server to udp.

**Pull request merged**